---a/net/minecraft/util/IStringSerializable.java
+++b/net/minecraft/util/IStringSerializable.java
@@ -1,7 +1,7 @@
 package net.minecraft.util;
 
-public interface IStringSerializable
-{
+public interface IStringSerializable {
+
     String getName();
-}
+}
